internal/install/recipe_installer_guided.go

```diff
@@ -34,6 +34,12 @@ func (i *RecipeInstaller) guidedInstall(ctx context.Context, m *types.DiscoveryM
 		return err
 	}
 
+	// Warn the user that --skipInfra is only implemented for targetedInstall
+	if i.SkipInfra {
+		log.Warnf("--skipInfra is only applicable to targeted installation")
+		log.Warnf("To skip the Infrastructure Agent installation, use: newrelic install -n <recipe_name> --skipInfra")
```

Comment:
@jbeveland27 that 2nd log statement is confusing. We're in a guided install context, why do we talk about -n <recipe_name>. Also newrelic on linux is in /usr/loca/bin, not in a global path (different path on windows).
It might be better to remove that 2nd log statement. I'm not sure what value it gives the user to know how to skip the infra agent when the option is already used

Reply:
It's meant to be a pointer to help correct the user toward what "targeted install" means, similar to the (did you mean? ...) type of prompts CLIs sometimes offer to a user. Not a big deal to remove though.

Reply:
```suggestion
- log.Warnf("--skipInfra is only applicable to targeted installation")
- log.Warnf("To skip the Infrastructure Agent installation, use: newrelic install -n <recipe_name> --skipInfra")
+ log.Warnf("--skipInfra is only applicable to targeted installation. Run newrelic install --help for usage")
```

Reply:
I like this ☝🏼